corehq/toggles/__init__.py

```diff
@@ -1464,7 +1464,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
 
 CASE_DEDUPE = StaticToggle(
```

Comment:
do we want this to be a `FrozenPrivilegeToggle` so that new users can't be added to it?

Reply:
> do we want this to be a `FrozenPrivilegeToggle` so that new users can't be added to it?

Thanks for pointing this class out to me. The immediate problem is that I don't believe there is an associated privilege for dedupe, which I'm making sure isn't an oversight on my part. Beyond that, I'm not sure this is a great fit. It sounds like `FrozenPrivilegeToggle` is meant to duplicate privilege functionality for domains that don't have the privilege. In this case, this toggle is meant to be for domains that are grandfathered into still allowing case updates, which the GA feature will not allow

Reply:
> this toggle is meant to be for domains that are grandfathered into...

I think this is exactly what [`FrozenPrivilegeToggle` was intended for](https://github.com/dimagi/commcare-hq/issues/32482#issuecomment-1398638587).

Reply:
Can you clarify further? From that CEP, my understanding is that GA'd features become privileges on software plans. Existing domains that use the GA'd functionality may not belong to those software plans, and so `FrozenPrivilegeToggle` is intended to bridge that gap. But, whether it is the privilege or the toggle, both are supporting the exact same workflow.

That last part is why I don't believe `FrozenPrivilegeToggle` is a good fit here. We are discussing different functionality: the GA'd feature will not allow updates, while the existing toggle still will (in fact, the toggle's sole purpose will be to support updates). I think its also reasonable to suspect that this update functionality will be open to domains being added and removed, because while we're GA-ing 75% of the feature, I imagine Solutions still wants to support clients that require 100% of the feature.